pub_sub/csharp/checkout/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Dapr.Client;
+using System.Threading;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+
+while(true) {
+    Random random = new Random();
+    var order = new Order(random.Next(1,1000));
+    //var data = JsonSerializer.Serialize<Order>(order);
+    CancellationTokenSource source = new CancellationTokenSource();
+    CancellationToken cancellationToken = source.Token;

```

Comment:
It would be better to just omit the cancellation token. Most users won't use one most of the time. 

This example isn't doing anything with the token so it should be simplified.

Reply:
I totally agree with you.  It's gone.